<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Attacksequence</class>
 <widget class="QDialog" name="Attacksequence">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Attack sequence</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_9">
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="2" column="1">
      <widget class="QLineEdit" name="defenseShipLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Unit:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="defenseUnitLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Ship:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="1">
      <widget class="QLineEdit" name="attackShipLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Unit:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="attackUnitLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Ship:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Defense dices</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Evade:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="evadeSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="rollDefensePushButton">
          <property name="text">
           <string>Roll!</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Attack results</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Hit:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="hitSpinBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton_5">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Reroll</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Critical hit:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="criticalSpinBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_6">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Reroll</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Focus:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="focusAttackSpinBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButton_7">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Reroll</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Blank:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="blankAttackSpinBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="pushButton_8">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Reroll</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Modifiers</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QListWidget" name="attackModListWidget">
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectItems</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="useAttackModPushButton">
        <property name="text">
         <string>Use modifier</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string>Modifiers</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QListWidget" name="defenseModListWidget"/>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="useDefenseModPushButton">
        <property name="text">
         <string>Use modifier</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Attack dices</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Attack:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="attackSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="rollAttackPushButton">
          <property name="text">
           <string>Roll!</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Defense results</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="1" column="1">
       <widget class="QSpinBox" name="focusDefenseSpinBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Evade:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="evadeDefenseSpinBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton_9">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Reroll</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Focus:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_11">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Reroll</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Blank:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="blankDefenseSpinBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButton_12">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Reroll</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Attacksequence</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Attacksequence</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rollAttackPushButton</sender>
   <signal>clicked()</signal>
   <receiver>Attacksequence</receiver>
   <slot>rollAttackDices()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rollDefensePushButton</sender>
   <signal>clicked()</signal>
   <receiver>Attacksequence</receiver>
   <slot>rollDefenseDices()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>448</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useAttackModPushButton</sender>
   <signal>clicked()</signal>
   <receiver>Attacksequence</receiver>
   <slot>useAttackModifier()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useDefenseModPushButton</sender>
   <signal>clicked()</signal>
   <receiver>Attacksequence</receiver>
   <slot>useDefenseModifier()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>Attacksequence</receiver>
   <slot>hitReroll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>Attacksequence</receiver>
   <slot>criticalHitReroll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>Attacksequence</receiver>
   <slot>attackFocusReroll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>Attacksequence</receiver>
   <slot>attackBlankReroll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_9</sender>
   <signal>clicked()</signal>
   <receiver>Attacksequence</receiver>
   <slot>evadeReroll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>448</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_11</sender>
   <signal>clicked()</signal>
   <receiver>Attacksequence</receiver>
   <slot>evadeFocusReroll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>448</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_12</sender>
   <signal>clicked()</signal>
   <receiver>Attacksequence</receiver>
   <slot>evadeBlankReroll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>448</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>rollAttackDices()</slot>
  <slot>rollDefenseDices()</slot>
  <slot>useAttackModifier()</slot>
  <slot>useDefenseModifier()</slot>
  <slot>hitReroll()</slot>
  <slot>criticalHitReroll()</slot>
  <slot>attackFocusReroll()</slot>
  <slot>attackBlankReroll()</slot>
  <slot>evadeReroll()</slot>
  <slot>evadeFocusReroll()</slot>
  <slot>evadeBlankReroll()</slot>
 </slots>
</ui>
